DQM/DTMonitorModule/src/DTLocalTriggerLutTask.h

```diff
@@ -86,11 +87,14 @@ class DTLocalTriggerLutTask: public DQMEDAnalyzer{
   bool detailedAnalysis;
   bool overUnderIn;
 
-  edm::EDGetTokenT<L1MuDTChambPhContainer> tm_Token_;
+  edm::EDGetTokenT<L1MuDTChambPhContainer> tm_TokenIn_;
+  edm::EDGetTokenT<L1MuDTChambPhContainer> tm_TokenOut_;
   edm::EDGetTokenT<DTRecSegment4DCollection> seg_Token_;
 
-  int trigQualBest[6][5][13];
-  const L1MuDTChambPhDigi* trigBest[6][5][13];
+  int trigQualBestIn[6][5][13];
```

Comment:
best to define these as arrays, then you get bounds checking

typedef DTArr3 std::array<std::array<std::array<int,13>, 5 > ,6> 

then trigQualBestIn[i][j][k] still works and you get the benefit of knowing how big arrays are throughout the code